### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by reemsh2000** [GSG-G11/news-team-2#57]
input tag should have a label for accessibility purposes and if you don't want the label text to be printed on the screen you can use the attributes like aria-label or title
